waku/v2/node/storage/message/sqlite_store/queries.nim

```diff
@@ -57,14 +58,14 @@ proc queryRowPubsubTopicCallback(s: ptr sqlite3_stmt, pubsubTopicCol: cint): str
 
 proc createTableQuery(table: string): SqlQueryStr = 
   "CREATE TABLE IF NOT EXISTS " & table & " (" &
-  " id BLOB," &
-  " storedAt INTEGER NOT NULL," &
-  " contentTopic BLOB NOT NULL," &
   " pubsubTopic BLOB NOT NULL," &
+  " contentTopic BLOB NOT NULL," &
   " payload BLOB," &
   " version INTEGER NOT NULL," &
-  " senderTimestamp INTEGER NOT NULL," &
-  " CONSTRAINT messageIndex PRIMARY KEY (senderTimestamp, id, pubsubTopic)" &
+  " timestamp INTEGER NOT NULL," &
```

Comment:
What does `timestamp` indicate? as per my understanding of the discussion around senderTimestamps, should this field be nullable?

Reply:
I thought the same, but when the field is not present in the protobuffer, it is equal to zero. So it won't be null.

BTW, I renamed the column to match the waku message field for consistency reasons.

Reply:
Ah! okay. Can we then make this field nullable, and if `timestamp` == 0, set it to null (would be 1 byte vs 4 bytes)?

Reply:
This can be completed in a follow up PR rather, ignore!